### ISSUE TITLE
Fix Request for UnconditionalCreatedLogArgument

### ISSUE THREAD

**Issue by jborgers** [jborgers/PMD-jPinpoint-rules#312]
example:
```
private void calcYear(Data data) {        
        Long loss = this.fService.filter(data);
        log.debug("loss for year={}, id={}, loss={}", data.getYear(), data.getId(), loss); // false positive on 'loss'
        this.lossService.storeLoss(data, loss.intValue());
}
```
the .intValue() is probably the culprit.

**Comment by jborgers** [comment]
Another example:

```
public void handle(final OrderDeleted event) {

   String group = service.getUserGroup(event.getOwner().getCode());
   final String key = event.getOwner().getCode()+ "#" + group + "#order";

   log.info("Preparing to remove order with key " + key); // false positive: Avoid unconditional creation of a log argument, it may not be needed.
 
   this.deleteFromRepository(key);
   log.info("Deleted order intent from repository: key " + key); // false positive: Avoid unconditional creation of a log argument, it may not be needed.
}
```

**Comment by jborgers** [comment]
The this. and the .intValue are both a problem for the rule
